test/e2e/storage/testsuites/base.go

```diff
@@ -286,10 +286,14 @@ func (r *genericVolumeTestResource) cleanupResource() {
 			}
 			if r.pvc != nil {
 				err := e2epv.DeletePersistentVolumeClaim(f.ClientSet, r.pvc.Name, f.Namespace.Name)
-				framework.ExpectNoError(err, "Failed to delete PVC %v", r.pvc.Name)
+				if err != nil {
+					return err
```

Comment:
PR looks good overall, I just have one suggestion: instead of `return err`, can you use `github.com/pkg/errors.Wrap` (https://github.com/pkg/errors#adding-context-to-an-error) to add more information about what went wrong and where?

The error itself might just be "timeout" - without additional information, it will be impossible to tell which operation timed out.

Can you check what error reporting via `framework.ExpectNoError(errors.NewAggregate(errs), "while cleaning up resource")` will then print?

`github.com/pkg/errors` captures the full stack trace for the call chain, but it might not be visible in the output.

Reply:
Due to errors.NewAggregate(errs), full stack trace can not be visible in the output. But using errors.wrap sill can help a lot, it can take a shot message when it happens. 

I have updated the PR, do you have other suggestions?